opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanShim.java

```diff
@@ -74,25 +78,25 @@ public Span setTag(String key, Number value) {
 
   @Override
   public Span log(Map<String, ?> fields) {
-    // TODO
+    span.addEvent(getEventNameFromFields(fields), convertToAttributes(fields));
     return this;
   }
 
   @Override
   public Span log(long timestampMicroseconds, Map<String, ?> fields) {
-    // TODO
+    span.addEvent(getEventNameFromFields(fields), convertToAttributes(fields));
```

Comment:
nit: perhaps this can be made in a single loop thru the attributes for perf.

Reply:
Well, that single-key check (when we look up for `event`) is not a biggie in terms of performance, I think ;)